build.gradle.kts

```diff
@@ -104,3 +105,10 @@ tasks.spotbugsMain {
         setStylesheet("fancy-hist.xsl")
     }
 }
+tasks.spotbugsTest {
```

Comment:
quante volte devi fare sto spotbugs?

Reply:
The new one is because spot bugs was failing on tests but the output was an xml file